src/h2load.cc

```diff
@@ -252,6 +260,56 @@ void rate_period_timeout_w_cb(struct ev_loop *loop, ev_timer *w, int revents) {
 } // namespace
 
 namespace {
+  // Called when the warmup duration for infinite number of requests are over
+  void warmup_timeout_cb(struct ev_loop *loop, ev_timer *w, int revents) {
+    auto client = static_cast<Client *>(w->data);
+    
+    if (client->worker->current_phase != MAIN_DURATION) {
```

Comment:
Yes, because multiple clients will have the warm up timeout, however, we will only acknowledge the first client.